data/Abdulwahid7027.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "Shaik Abdul Wahid",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Student • Frontend developer • Open Source Enthusiast • Python • Java",
+  "avatar": "https://avatars.githubusercontent.com/u/110891341?s=96&v=4",
```

Update the avatar url to short link.
```suggestion
  "avatar": "https://github.com/Abdulwahid7027.png",
```